src/components/mint-button-primary/mint-button-primary.spec.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { availableTypes as types, default as Button} from './index';
+import $ from 'teaspoon';
+
+describe('mint-button-primary', () => {
+  it('should have defaultTypes available', () => {
+    expect(types).toBeDefined();
+  });
+
+  it('should have dark type available', () => {
+    expect(types.dark).toBeDefined();
+  });
+
+  it('should have alt type available', () => {
+    expect(types.alt).toBeDefined();
+  });
+
+  it('should have mint-button-primary class', () => {
+    expect($(<Button/>).render()).toHaveClass('mint-button-primary');
```

Comment:
The output should match BEM structure we have right now.
This should be checked in tests for all components.